src/train_model.py

```diff
@@ -1,17 +1,25 @@
-# This is for training a model
+import pickle
+import pandas as pd
 from sklearn.model_selection import train_test_split
-from sklearn.tree import DecisionTreeClassifier
+from sklearn.linear_model import LogisticRegression
 from sklearn.metrics import accuracy_score
+import config
 
-X, y = load_dataset()
+df = pd.read_csv(config.SOURCE)
 
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+X = df.drop([config.TARGET], axis=1)
+y = df[config.TARGET]
 
-model = DecisionTreeClassifier()
+X_train, X_test, y_train, y_test = train_test_split(
```

Comment:
should we have a naming convention? for example, X_train starts with an uppercase, but y_train with a lowercase. In my opinion, variables should always begin with a lowercase, unless  they are constants or classes